examples/kafka-hub/hub/hub_service.bal

```diff
@@ -53,9 +53,6 @@ service object {
     #            if topic registration failed or `error` if there is any unexpected error
     isolated remote function onRegisterTopic(readonly & websubhub:TopicRegistration message, http:Headers headers)
                                 returns websubhub:TopicRegistrationSuccess|websubhub:TopicRegistrationError|error {
-        if config:SECURITY_ON {
-            check security:authorize(headers, message.topic);
```

Comment:
At which point the `security:authorize()` is called if this is removed?

Reply:
Since, at the moment we are not using any JWT based auth/authz mechanism from publisher (asgardeo) side, removed this.